src/Time/Time.php

```diff
@@ -16,118 +16,111 @@
 
 namespace Zest\Time;
 
+use Zest\Common\TimeZone;
 use Zest\Contracts\Time\Time as TimeContract;
 
 class Time implements TimeContract
 {
     /**
-     * Get "now" time
+     * Get "now" time.
      *
      * @param string $timezone Valid php supported timezone.
      *
      * @since 3.0.0
      *
-     * @return  int
+     * @return int
      */
-    public static function now($timezone = NULL)
+    public static function now($timezone = '')
     {
+        TimeZone::seteDefaultTz($timezone);
 
+        return time();
     }
 
     /**
-     * Number of days in a month
+     * Number of days in a month.
      *
      * @param int $month A numeric month.
      * @param int $year  A numeric year.
      *
      * @since 3.0.0
      *
-     * @return  int
+     * @return int
      */
     public static function daysInMonth($month = 1, $year = 1970)
     {
-
+        return cal_days_in_month(CAL_GREGORIAN, $month, $year);
```

Comment:
Using the `cal_days_in_month` function which is provided by `calendar` extension in PHP to calculate the days of given month.